### ISSUE TITLE
Authentik setup using default instructions on documentation website does not work on http endpoint

### ISSUE THREAD

**Issue by arpanghosh8453** [goauthentik/authentik#8054]
**Describe the bug**

I found a [similar post](https://www.reddit.com/r/selfhosted/comments/18jnu4q/assistance_with_authentik/) here. I followed the same tutorial for the most recent version given in the docker-comose instllation section. I discovered that the HTTP endpoint could not be used to do the initial setup or login. The HTTP endpoint opens but does not let you go to the password phase ( even for the default akadmin user ), and no errors are shown on container logs. ( http code is 200 for the request ). If you start the setup ( initial password creation ) from the https endpoint, everything works fine. you can create the password and log in. Nowhere in the authentic documentation is it mentioned that the HTTP endpoint cannot be used, and the documentation instructions say to go to http://server-ip:9000/if/flow/initial-setup/. This is a bug. Anyone facing the same issue knows why that is the case.

**To Reproduce**
Follow the docker compose setup (https://goauthentik.io/docs/installation/docker-compose) to create a fresh install. go to the http endpoint (http://server-ip:9000/if/flow/initial-setup/) and try creating the user. It does not work. And if a user already exists, go to the login page over the http endpoint, and try login. both of these work fine over https endpoint

**Expected behavior**
Both http and https endpoints ( ports ) should work the same. 

**Version and Deployment (please complete the following information):**

-   authentik version: 2023.10.5
-   Deployment: docker-compose

**Comment by rissson** [comment]
Just tried this:
```
wget https://goauthentik.io/docker-compose.yml
echo "PG_PASS=$(pwgen -s 40 1)" >> .env
echo "AUTHENTIK_SECRET_KEY=$(pwgen -s 50 1)" >> .env
echo "AUTHENTIK_ERROR_REPORTING__ENABLED=true" >> .env
docker-compose pull
docker-compose up -d
```
Went to http://localhost:9000, and refreshed until authentik was up.
Then went to http://localhost:9000/if/flow/initial-setup/ and refreshed until the Not found error was not there. It may take a bit for the initial setup to be ready, as authentik is setting things up on the first run and that may take some time.
But I can confirm that the instruction in the documentation do work. Perhaps you're having some other issue.
Please look at the server and worker logs (`docker-compose logs server` and `docker-compose logs worker`) and see if anything is out of place.

**Comment by arpanghosh8453** [comment]
Thank you for taking a look into it. On my system, the page loads on http. I can reach the initial setup pages too. But can't do the login or registration over the http port. If I use the default password setup for akadmin, I can login fine over https after everything starts. The http port login will still not work.

I checked the docker logs, no errors were there ( server and worker). It registers the http request and return 200 but it does not go to the password stage on http port.

No big deal, I use https anyway, but just thought I should notify the devs here. The project is awesome! And I use it for almost everything on my self hosted network. I am not sure why I am having this issue. I did Authentik setup before and things worked fine.
